docs/docs.jsx

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 import Ecology from "ecology";
 import Radium, { Style } from "radium";
-
 import { VictoryTheme } from "formidable-landers";
+import * as docgen from "react-docgen"
```

Comment:
I assume there is no default export from `react-docgen` so you can do `import docgen from "react-docgen"?
You [_could_ do](https://github.com/reactjs/react-docgen/blob/master/src/main.js#L56) `import { parse as parseComponent } from "react-docgen"` (or whatever alias you want to use) instead; I don't feel strongly either way.